### PR TITLE
fix build with latest data-default

### PR DESCRIPTION
**Pull request by abuibrahim**

### CODE REVIEW

File: Cheapskate/Util.hs [hunk 1]
```diff
@@ -21,7 +21,6 @@ module Cheapskate.Util (
 import Data.Text (Text)
 import qualified Data.Text as T
 import Data.Char
-import Control.Applicative
```
Comment (jgm) [inline Cheapskate/Util.hs]: I doubt this is redundant on older versions of base.

**Comment by abuibrahim** [comment]
You're right. How about importing it conditionally?

**Comment by jgm** [comment]
That would be fine.

+++ Ruslan Babayev [Sep 23 16 09:36 ]:

**Comment by jgm** [comment]
Thanks for the contribution (very belatedly merged)!

**Pull request merged**